metadata/002/cip72.json

```diff
@@ -3,7 +3,7 @@
   "projectName": "FakeRayWallet 003",
   "link": "https://fakeraywallet.testdapp.x/app",
   "logo": "https://github.com/lucadonazzon/cip72/blob/main/metadata/002/DJAUs3nJ_400x400.jpg",
-  "category": "DeFi",
+  "categories": "DeFi",
```

Comment:
It should be an array. 
```suggestion
  "categories": ["DeFi"],
```

Reply:
We also don't need this change, I'll revert my changes to this file and make all metadata changes in the ticket dedicated to it.